src/convnets/resnet.jl

```diff
@@ -175,6 +197,35 @@ See also [`Metalhead.resnet`](#).
 
 !!! warning
     Only `ResNet(50)` currently supports pretrained weights.
+
+For `ResNet(18)` and `ResNet(34)`, the parameter-free shortcut style (type `:A`)
+is used in the first block and the three other blocks use type `:B` connection
+(following the implementation in PyTorch). The published version of
+`ResNet(18)` and `ResNet(34)` used type `:A` shortcuts for all four blocks. The
+example below shows how to create a 18 or 34-layer `ResNet` using only type `:A`
+shortcuts:
+
+```julia
+using Metalhead
+
+resnet18 = ResNet([1, 1], [2, 2, 2, 2], :A;
+    block = Metalhead.basicblock)
+
+resnet34 = ResNet([1, 1], [3, 4, 6, 3], :A;
+    block = Metalhead.basicblock)
+```
+
+The bottleneck of the orginal ResNet model has a stride of 2 on the first
+convolutional layer when downsampling (instead of the 2nd convolutional layers
+as in ResNet v1.5). The architecture of the orignal ResNet model can be obtained
+as shown below:
+
+```julia
+bottleneck_v1(inplanes, outplanes, downsample = false) =
+    Metalhead.bottleneck(inplanes, outplanes, downsample; stride = [(downsample ? 2 : 1), 1, 1])
```

Comment:
Can we just add a function like this to the code base? Then here we just say "use `Metalhead.bottleneck_v1` for the original behavior.

Reply:
ok, no problem.